flatten-me.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""Flatten a list of strings and lists into a new list of discrete elements."""
+
+
+def flatten_me(lst):
+    """Parse elements in a list and append a new list to return."""
+    idx = 0
+    new_list = []
+    while idx < len(lst):
+        if type(lst[idx]) == list:
+            j = 0
```

Comment:
These lines of code can be easily replaced with the very useful `list.extend` method. Please read the docs on `list.extend` and make sure you know how to use it, because it will save you a lot of lines of code!